src/platform/testing/unit/mocha-setup.js

```diff
@@ -76,6 +76,10 @@ function setupJSDom() {
     url: 'http://localhost',
   });
 
+  if (global.fetch.isSinonProxy) {
```

Comment:
This isn't my place to say, since this is clearly VSP code, but it feels a little wrong to do this bit as part of a `setupJSDom()` function since this doesn't seem to have anything to do with setting up JSDOM.

Reply:
That's a good call. I'm happy to move it out into it's own function.

Reply:
Done. Thanks! :)